web/libs/editor/src/components/Timeline/Views/Wave/Wave.scss

```diff
@@ -93,7 +93,7 @@
         position: absolute;
         top: -16px;
         left: -4px;
-        content: url("data:image/svg+xml,%3Csvgwidth='9'height='5'viewBox='0095'fill='none'xmlns='http://www.w3.org/2000/svg'%3E%3Cpathd='M11C10.7238581.223860.51.50.5H7.5C7.776140.580.72385881V3.21922C83.448667.843853.648657.621273.70429L4.621274.4543C4.541654.47424.458354.47424.378734.45429L1.378733.70429L1.257464.18937L1.378733.70429C1.156153.6486513.4486613.21922V1Z'fill='%2340A9FF'stroke='%231F1F1F'/%3E%3C/svg%3E%0A");
+        content: url("data:image/svg+xml;charset=utf-8;base64,PHN2ZyB3aWR0aD0nOSdoZWlnaHQ9JzUndmlld0JveD0nMDA5NSdmaWxsPSdub25lJ3htbG5zPSdodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2Zyc+PHBhdGggZD0nTTExQzEwLjcyMzg1ODEuMjIzODYwLjUxLjUwLjVINy41QzcuNzc2MTQwLjU4MC43MjM4NTg4MVYzLjIxOTIyQzgzLjQ0ODY2Ny44NDM4NTMuNjQ4NjU3LjYyMTI3My43MDQyOUw0LjYyMTI3NC40NTQzQzQuNTQxNjU0LjQ3NDI0LjQ1ODM1NC40NzQyNC4zNzg3MzQuNDU0MjlMMS4zNzg3MzMuNzA0MjlMMS4yNTc0NjQuMTg5MzdMMS4zNzg3MzMuNzA0MjlDMS4xNTYxNTMuNjQ4NjUxMy40NDg2NjEzLjIxOTIyVjFaJ2ZpbGw9JyM0MEE5RkYnc3Ryb2tlPScjMUYxRjFGJy8+PC9zdmc+");
```

Comment:
What is the relation to Tailwind and this change? Looks correct, just curious why it was changed here.

Reply:
pushed accidentally, removed